spec/controllers/events_controller_spec.rb

```diff
@@ -3,4 +3,17 @@
 require 'rails_helper'
 
 RSpec.describe EventsController, type: :controller do
```

Comment:
We are missing tests to create an event, because #new just gets the form but doesn't create an event. Furthermore, we need tests for the rest of the CRUD operations. Event management means being able to manage an event(s), which implies doing create, read, update and delete operations. These all ought to be tested.

For create you need to do a POST request, for a read operation, you need to use a GET request and then checking to see that the event you've accessed (via :show) is an event you previously created (in the same test). Look up what type of requests you need to do to test the remaining ones: Update and Delete, and test that an event can be updated and deleted.

[`REQUIRED`]